mqtt-sparkplug-plus.html

```diff
@@ -101,6 +101,7 @@
                     <option value="">No</option>
                     <option value="DEFLATE">Deflate</option>
                     <option value="GZIP">Gzip</option>
+                    <option value="GZIP">Gzip</option>
                 </select>
```

Comment:
this line seems redundent?

Reply:
Yeah seems like I ve made a mistake. I will remove this